docker-compose.dev.yml

```diff
@@ -37,6 +37,17 @@ services:
     volumes_from:
       - localdev
     entrypoint: ['bundle']
+    links:
```

Comment:
We should create a new application entry using `bundle` as an example, changing the name and entry point to `erd`. This should allow us to call `docker-compose run erd` without having to use `bundle exec`.